src/matter/cluster/LevelControlCluster.ts

```diff
@@ -55,23 +55,23 @@ const MoveToClosestFrequencyCommandRequest = TlvObject({
 /** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.6.4 */
 const features = {
     /** Dependency with the On/Off cluster */
-    onOffClusterDependency: BitFlag(0), // default: true
+    onOff: BitFlag(0), // default: true
 
     /** Behavior that supports lighting applications. */
     lighting: BitFlag(1),
 
     /** Supports frequency attributes and behavior. */
-    supportFrequency: BitFlag(2)
+    frequency: BitFlag(2)
```

Comment:
I originally had this as "frequency" and also "onOff" as in the spec, but changed it based on comment from mfucci who wanted these to be more descriptive.

Reply:
We should be close to specs and XML (this is also something we agreed on. With the comment line over it it should be clear also in IDE when using it

Reply:
I'd tend to agree that unless there is a technical reason not to, we should stick to the spec even when it could have been written better.

Reply:
OK.... I asked to rename those since the spec names were pretty bad, but fine...